src/GameSelector/UserGame.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box, HStack, Text, VStack } from 'native-base';
+import React from 'react';
+import { Pressable } from 'react-native';
+import { Board } from '../GameState/Board';
+import { Game } from '../interface';
+
+interface UserGameProps {
+  navigateToGameState: (id: number | undefined) => void;
+  game: Game | undefined;
+}
+
+const GRADIENT_COLORS: any = [];
+GRADIENT_COLORS[0] = {
+  background: ['orange.400', 'amber.400'],
+  foreground: 'black',
+};
+GRADIENT_COLORS[1] = {
+  background: ['emerald.400', 'lime.400'],
+  foreground: 'black',
+};
+GRADIENT_COLORS[2] = {
+  background: ['error.600', 'danger.300'],
+  foreground: 'black',
+};
+
+export const UserGame = ({ game, navigateToGameState }: UserGameProps) => {
+  const getGradientColor = (bool: boolean) => {
+    if (bool) {
+      return GRADIENT_COLORS[1];
+    }
+    return GRADIENT_COLORS[0];
+  };
```

Comment:
Two things about this method.

1. It's a generic method that should be moved away from this component.
2. The name bool is not explicit at all. You should give meaning to your variables.